backend/populateDB/schema.sql

```diff
@@ -457,6 +460,11 @@ $$ language plpgsql strict security definer;
 
 comment on function floods.new_crossing(text, text, text, integer, decimal, decimal) is 'Adds a crossing.';
 
+create function floods.human_cooordinates(crossing floods.crossing) returns text as $$
+  select ST_AsLatLonText(floods.crossing.coordinates) from floods.crossing
```

Comment:
hmm, I think we should be using crossing (the param) instead of floods.crossing (the table) here
`select ST_AsLatLonText(crossing.coordinates) `